src/main/java/mServer/crawler/sender/MediathekMdr.java

```diff
@@ -99,25 +101,27 @@ public void addToList() {
             if (pos1 != -1 && pos2 != -1) {
                 url = seite.substring(pos1, pos2);
             }
-            if (url.equals("")) {
+            if (url.isEmpty()) {
                 Log.errorLog(461225808, "keine URL");
             } else {
                 url = MUSTER_ADD_TAGE + url;
+                //FIXME
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)